docs/integrations/prometheus.md

```diff
@@ -6,28 +6,65 @@ This document is a getting started guide to integrating M3DB with Prometheus.
 
 To write to a remote M3DB cluster the simplest configuration is to run `m3coordinator` as a sidecar alongside Prometheus.
 
-Start by downloading the [config template](https://github.com/m3db/m3db/blob/master/src/coordinator/config/m3coordinator-cluster-template.yml). Update the config ‘service’ and 'seedNodes' sections to match your cluster's configuration, something like:
-
-```
-config:
-  service:
-    env: default_env
-    zone: embedded
-    service: m3db
-    cacheDir: /var/lib/m3kv
-    etcdClusters:
-      - zone: embedded
-        endpoints:
-          - SEED_HOST_1_STATIC_IP:2379
-          - SEED_HOST_2_STATIC_IP:2379
-          ...
-  seedNodes:
-    initialCluster:
-      - hostID: SEED_HOST_1_NAME
-        endpoint: http://SEED_HOST_1_STATIC_IP:2380
-      - hostID: SEED_HOST_2_NAME
-        endpoint: http://SEED_HOST_2_STATIC_IP:2380
-      ...
+Start by downloading the [config template](https://github.com/m3db/m3db/blob/master/src/coordinator/config/m3coordinator-cluster-template.yml). Update the `namespaces` and the `client` section for a new cluster to match your cluster's configuration.
+
+At the very least you'll need to specify the name of your namespace that you setup, the retention that it has.  You can leave the metrics type as `unaggregated` since it's required by default to have a cluster that receives all Prometheus metrics unaggregated.  In the future you might also want to aggregate and downsample metrics for longer retention, and you can come back and update the config once you've setup those clusters.
```

Comment:
The first line of this and the next line can be combined into something like `You'll need to specify the static IPs or hostnames of your M3DB seed nodes, and the name and retention values of the namespace you set up.`

Reply:
Sure thing, done.